frontend/public/components/storage-class-form.tsx

```diff
@@ -103,6 +103,44 @@ export class StorageClassForm_ extends React.Component<
   // For 'csi' storage type
   CSIStorageTypes = Object.freeze({
     ...this.getExtensionsStorageClassProvisioners(Provisioner.CSI),
+    'ebs.csi.aws.com': {
+      title: 'AWS CSI',
+      provisioner: 'ebs.csi.aws.com',
+      allowVolumeExpansion: true,
+      parameters: {
+        type: {
+          name: 'Type',
+          values: { io1: 'io1', gp2: 'gp2', sc1: 'sc1', st1: 'st1', standard: 'standard' },
+          hintText: 'Select AWS Type. Default is gp2',
+        },
+        iopsPerGB: {
+          name: 'IOPS Per GiB',
+          hintText: 'I/O operations per second per GiB',
+          validation: (params) => {
+            if (params.iopsPerGB.value !== '' && !params.iopsPerGB.value.match(/^[1-9]\d*$/)) {
```

Comment:
Isn't this the same as `^\d+$`? Are fractional values allowed? (If so, the regex is probably wrong.)

```suggestion
            if (params.iopsPerGB.value && !params.iopsPerGB.value.match(/^\d+$/)) {
```

Reply:
`Number.isFinite(Number(params.iopsPerGB.value))` might be better. Although it looks like you've copied what's below.

Reply:
`(/^\d+$/)` && `(/^\d+$/)` is same and fractional values are not allowed looking at code[1].

[1] . https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/44ec7de2c953b52c9fd39e85ca2183aab6465d32/pkg/driver/controller.go#L128

`Number.isFinite(Number(params.iopsPerGB.value))` should I change it at both places?

Reply:
`/^\d+$/` is good if we don't allow fractions.